src/data/polynomial/degree/basic.lean

```diff
@@ -2,6 +2,7 @@
 Copyright (c) 2018 Chris Hughes. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Chris Hughes, Johannes Hölzl, Scott Morrison, Jens Wagemaker
+Co-authored-by: Damiano Testa <adomani@gmail.com>
```

Comment:
The `Co-authored-by:` is only for git commit messages (if there is a coauthor).
In this case, simply add your name to the line above.

Reply:
Should it follow the convention that is alphabetical order, or should it be chronological order?

Reply:
@jcommelin I do not know what happened with my git.  I was using branches and I do not understand them.  After trying for hours to pull, push, rebase, fetch, merge, I do not know how I created a new pull request that I hope has these modifications that you suggested.  It is #4509.  Sorry for creating more confusion!